infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackSpatialCache.kt

```diff
@@ -67,7 +67,7 @@ constructor(
 data class SpatialCacheSegment(
     val locationTrackVersion: LayoutRowVersion<LocationTrack>,
     val alignmentVersion: RowVersion<LayoutAlignment>,
-    val segment: LayoutSegment,
+    val segmentIndex: Int,
 )
```

Comment:
Tuntuu hieman oudolta että kakun segmenttioliosta vaihdetaan segmenttireferenssi pelkäksi indeksiksi (luulisi että jos tämä on kannattavaa niin ihan koko kakussa, tai ainakin segmenttien kakuttamisessa, on jotain mätää) 🤔 Mutta tämä on ihan puhtaasti mutua, voi olla että tämä ei olekaan haisu mistään muusta ongelmasta. Nimeäisin tosin tuon `SpatialCacheSegment`:in nyt joksikin muuksi, joka ehkä kertoisi vähän paremmin että se ei enää sisällä koko segmenttiä vaan ainoastaan ohjeet sen löytämiseen (esim. `SpatialCacheSegmentReference` tms.)